lib/components/AlbumScreen/album_screen_content.dart

```diff
@@ -109,18 +109,23 @@ class _AlbumScreenContentState extends State<AlbumScreenContent> {
                 ),
                 sliver: SongsSliverList(
                   childrenForList: childrenOfThisDisc,
-                  childrenForQueue: Future.value(widget.children),
+                  childrenForQueue: Future.value(widget.queueChildren),
                   parent: widget.parent,
                   onRemoveFromList: onDelete,
                 ),
               )
-          else if (widget.children.isNotEmpty)
+          else if (widget.displayChildren.isNotEmpty)
             SongsSliverList(
-              childrenForList: widget.children,
-              childrenForQueue: Future.value(widget.children),
+              childrenForList: widget.displayChildren,
+              childrenForQueue: Future.value(widget.queueChildren),
               parent: widget.parent,
               onRemoveFromList: onDelete,
             ),
+          SliverToBoxAdapter(
+            child: Container(
+              height: MediaQuery.paddingOf(context).bottom,
+            ),
+          )
```

Comment:
extendbody: true extends the scroll areas under the now playing bar.  This allows everything to scroll past the bar cleanly.  MediaQuery.paddingOf(context).bottom is set by the scaffold to the height of whatever is in the bottom slot.  The music screen works without needing this, possibly because it isn't a sliver list like the artist/album screens.

Reply:
Ah I see. Does this also reapect the height of the snackbars? I noticed that they end up blocking the bottom items on the music screen, so I might have to look at applying that solution there as well.  
Thanks for the explanation!

Reply:
It looks like it doesn't care about snackbars, only the item in bottomNavigationBar.

Reply:
Okay. But I guess that's a problem for a different PR ^^